stdlib/stdlib.mli

```diff
@@ -892,8 +892,12 @@ val prerr_newline : unit -> unit
 
 val read_line : unit -> string
 (** Flush standard output, then read characters from standard input
-   until a newline character is encountered. Return the string of
-   all characters read, without the newline character at the end. *)
+   until a newline character is encountered. 
+   
+   Return the string of all characters read, without the newline character
+   at the end. 
+   
+   @raise End_of_file if the end of the file is reached at the beginning of line.*)
```

Comment:
This line is too long (> 80 chars) I think. You should run `tools/check-typo` and make sure it passes.

Reply:
Corrected.  This was an opportunistic improvement, so I don't have a working tool to run `tools/check-typo` sorry.